src/console_cmds.cpp

```diff
@@ -469,7 +469,7 @@ DEF_CONSOLE_CMD(ConClearBuffer)
  * Network Core Console Commands
  **********************************/
 
-static bool ConKickOrBan(const char *argv, bool ban)
+static bool ConKickOrBan(const char *argv, bool ban, const char *reason = nullptr)
```

Comment:
I wonder whether this function (and the others) need to have the default parameters. Would be more consistent to explicitly pass nullptr if no reason, imo

Reply:
I agree that the functions related to kicking/banning a client should explicitly pass nullptr if no kick reason is supplied. However, the function SendError is used in more cases than just kicking a client, all of which don't support printing a message at the moment. So I'm not sure that removing the default parameter and explicitly passing nullptr every time that SendError is called would increase the readability of the code. What do you think?

Reply:
I agree that the functions related to kicking/banning a client should explicitly pass nullptr if no kick reason is supplied. However, the function SendError is used in more cases than just kicking a client, all of which don't support printing a message at the moment. So I'm not sure that removing the default parameter and explicitly passing nullptr every time that SendError is called would increase the readability of the code. What do you think?

Reply:
Yeah, that makes sense. Didn't notice that that also applied to SendError